### PR TITLE
behat: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/behat.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# behat
+
+> A PHP framework for automatically testing business expectations.
```
Comment (andrik) [inline pages/common/behat.md]: Behavior-Driven Development framework for PHP

Comment (sbrl) [reply pages/common/behat.md]:
```suggestion
> Behavior-Driven Development framework for PHP.
```

Comment (agnivade) [reply pages/common/behat.md]: Another: `A PHP framework for Behavior-Driven Development`

**Comment by sbrl** [comment]
Are we ok to merge, @andrik?

**Pull request merged**